test_scripts/Resumption/InteriorVehicleData/007_InteriorVD_resumption_one_moduleType_with_moduleId_IGN_OFF.lua

```diff
@@ -0,0 +1,58 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: Successful resuming of interior vehicle data after IGN_OFF
+--  in case app was subscribed to one module with moduleId
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app with REMOTE_CONTROL hmi type is registered and activated
+-- 3. App is subscribed to module_1 via GetInteriorVehicleData(module_1, moduleId)
+--
+-- Sequence:
+-- 1. IGN_OFF and IGN_ON are performed
+-- 2. App starts registration with actual hashId after SDL restart
+-- SDL does:
+-- - a. send RC.GetInteriorVehicleData(subscribe=true, module_1, moduleId) to HMI during resumption data
+-- - b. respond RAI(SUCCESS) to mobile app
```

Comment:
please add 
b. SDL get a successful responses for request with isSubscribed=true
c. SDL respond RAI(success=true, result code = SUCCESS) to mobile app

Reply:
Done